app/enterprise/1.5.x/kong-for-kubernetes/install.md

```diff
@@ -158,13 +152,223 @@ $ export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip
+<img src="https://doc-assets.konghq.com/kubernetes/Kong-Enterprise-on-Kubernetes.png">
 
-To install Kong Enterprise on Kubernetes, instead of installing Kong for Kubernetes, see the following diagram. (Steps to be provided at a later time.)
+### Set up Helm
 
-<img src="https://doc-assets.konghq.com/kubernetes/Kong-Enterprise-on-Kubernetes.png">
+1. Add the Kong charts repository:
+    ```
+    $ helm repo add kong https://charts.konghq.com
+    ```
+2. Update Helm:
+    ```
+    $ helm repo update
+    ```
+
+### Configure Kong Enterprise Docker registry access
+Set up Docker credentials to allow Kubernetes nodes to pull down the Kong Enterprise Docker image, which is hosted as a private repository. You receive credentials for the Kong Enterprise Docker image when you sign up for Kong Enterprise.
+
+{% navtabs %}
+{% navtab kubectl %}
+1. Set up the credentials:
+    ```
+    $ kubectl create secret -n kong docker-registry kong-enterprise-edition-docker \
+        --docker-server=kong-docker-kong-enterprise-edition-docker.bintray.io \
+        --docker-username=<your-bintray-username> \
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    kubectl create secret generic kong-enterprise-superuser-password  -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% navtab OpenShift oc %}
+1. Set up the credentials:
+    ```
+    $ oc create secret -n kong docker-registry kong-enterprise-edition-docker \
+        --docker-server=kong-docker-kong-enterprise-edition-docker.bintray.io \
+        --docker-username=<your-bintray-username> \
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    oc create secret generic kong-enterprise-superuser-password -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% endnavtabs %}
+### Enable the sessions plugin for Manager and Dev Portal
+In the following steps, replace `<your-password>` with a secure password.
+
+{% navtabs %}
+{% navtab kubectl %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ cat > admin_gui_session_conf
+    {"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ cat > portal_session_conf
+    {"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+
+3. Create secret:
+    ```
+    $ kubectl create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
```

Comment:
We have seen that a lot of users run into issues when executing this command because they won't have the files name correctly. They will often have `.json` suffix in the filename. To get around this, I recommend instead using the following form:

```
kubectl create secret generic db-user-pass --from-file=username=./username.txt --from-file=password=./password.txt
```

Notice how `--from-file` is of the form `--from-file=<key-name>=<filename>`. This removes the problem I mentioned above.